### ISSUE TITLE
Possibility to change color for state buttons

### ISSUE THREAD

**Issue by raven2cz** [mut-ex/awesome-wm-nice#7]
Is is possible to change the color for state buttons (not just tooltip) in the awesomewm:  sticky mode, onTop mode, floating mode?
Changing of colors for hover mouse mode for close, minimize and maxime works nice ;-)

**Comment by mut-ex** [comment]
Yes all button colors can be changed through the following config params:

>   -- close_color = "#ee4266"
  -- minimize_color = "#ffb400"
  -- maximize_color = "#4CBB17"
  -- floating_color = "#f6a2ed"
  -- ontop_color = "#f6a2ed"
  -- sticky_color = "#f6a2ed"

However currently you can not specify different colors for on/off states (but it is planned)
